get-pipsi.py

```diff
@@ -5,6 +5,7 @@
 import sys
 from subprocess import call
 import textwrap
+from pipsi import get_real_python
```

Comment:
that import is not available for the intended use of that script, its only working in a checkout

Reply:
oops, you're right, just forgot that 😅

Reply:
fixed in the latest commit